packages/tendermint-rpc/types/index.d.ts

```diff
@@ -1,6 +1,7 @@
 export { Adaptor } from "./adaptor";
 export { adaptor33, adaptor34 } from "./adaptors";
 export { Client } from "./client";
+export { DateTime, DateTimeString } from "./encodings";
```

Comment:
Could we remove the type `DateTimeString` completely instead of exporting it? I think those type aliases never added enough value to justify them. It does not seem to be best practice in TypeScript, otherwise there would be a non-hacky solution.

Reply:
😢

Reply:
Do you like them?

Reply:
https://github.com/cosmos/cosmjs/issues/580

Reply:
> Do you like them?

I... like that they're more specific, but I think you're right that TypeScript doesn't really want you to do this.